src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs

```diff
@@ -112,7 +112,13 @@ public class BlockchainServiceWeb3 : JsonRpcService
 
             if(fullTx) 
                 txArray = Web3DataFormatUtils.Web3BlockTransactionArray(txs, block!.Hash, block!.Header.Index);
-
+            else
```

Comment:
For the future: do not place different fixes in one pull request,  create other pull request if you need to fix something not related,  pull requests should have a narrow focus,  it really simplifies a work.

Reply:
Okay, I will keep in mind.